### ISSUE TITLE
Spotify Cell expansion (revisited)

### ISSUE THREAD

**Issue by mzgaljic** [MortimerGoro/MGSwipeTableCell#96]
Making another Spotify cell issue   :-) 

So in issue #72, we figured out how to get the basic functionality to be the same as the Spotify style cells. The problem I am having now is that it is still possible to reveal the buttons as if they were normal cell buttons. The buttons should NEVER become exposed like that if one is trying to copy the Spotify cell look, hence my issue lol. I have recorded a gif to show you what I am talking about. If you are confused what I'm trying to show, please have a look at the gif I posted in #72 for comparison.

Would appreciate any suggestions.

Also please note that the issue here in my GIF is the left button becoming exposed. I am only trying to copy the Spotify look for the action "swipe left -> right"...if that makes sense. So disregard the delete button becoming exposed in the GIF, that is my desired behavior...focus on the grey "queue" button.
![](http://i.imgur.com/bfsCeSO.gif)

**Comment by MortimerGoro** [comment]
Hi again mzgaljic,

I've finally taken the time to work on a new release. The problem was related to some swipe inertia calculations which doesn't use the threshold value. I have added a new property called keepButtonsSwiped. You can set the property to false instead of setting a high threshold, try it ;)

It's available in develop branch

```
expansionSettings.fillOnTrigger = NO;
expansionSettings.threshold = 1;
expansionSettings.expansionLayout = MGSwipeExpansionLayoutCenter;
expansionSettings.expansionColor = [UIColor greenColor];
swipeSettings.transition = MGSwipeTransitionClipCenter;
swipeSettings.keepButtonsSwiped = NO;
```

**Comment by mzgaljic** [comment]
That is very cool! Thanks for the update...I'll check it out this weekend

**Comment by MortimerGoro** [comment]
I have published 1.5.1. I have created a Spotify demo too ;)

**Comment by mzgaljic** [comment]
Finally got a chance to take a look at this. It's great now, thank you!
